modules/network/src/test/java/org/apache/ignite/internal/network/netty/NettyServerTest.java

```diff
@@ -80,7 +83,10 @@ public class NettyServerTest extends BaseIgniteAbstractTest {
      */
     @AfterEach
     final void tearDown() throws Exception {
-        server.stop().join();
+        if (server != null) {
```

Comment:
I would also suggest to use `IgniteUtils.closeAll` here

Reply:
bootstrap/server are not autocloseable, and stopAll also can't be used because server is not an IgniteComponent

Reply:
To make something AutoCloseable, you can use a lambda: `AutoCloseable x = server == null ? null : server::stop`